packages/devextreme/js/ui/html_editor/ui.html_editor.js

```diff
@@ -189,7 +190,11 @@ const HtmlEditor = Editor.inherit({
         const frameDocument = frameWindow.document;
         const frameDocumentBody = frameDocument.body;
 
-        frameDocumentBody.innerHTML = value;
+        const quill = getQuill();
+
+        frameDocumentBody.innerHTML = quill.replaceStyleAttribute
```

Comment:
Hm, how `quill.replaceStyleAttribute` can not exist?

Reply:
it's temporary solution to avoid breaking before new version release. I added this because I think what we can merge this PR before release new version. Bu at present, it not possible. I will remove this condition

Reply:
removed